### ISSUE TITLE
what is the different between tf.mul and * when i used to build a graph

### ISSUE THREAD

**Issue by MisayaZ** [tensorflow/tensorflow#2804]
what is the difference between tf.mul and * when i used to build a graph?
Can both of them calculate the gradient automatically?

**Comment by ebrevdo** [comment]
There is no difference. * is sugar for tf.mul.

tf.mul does backpropagate gradients correctly.
